src/utilities/buildClientSchema.js

```diff
@@ -60,16 +60,7 @@ import type {
   IntrospectionNamedTypeRef,
 } from './introspectionQuery';
 
-type Options = {|
-  /**
-   * When building a schema from a GraphQL service's introspection result, it
-   * might be safe to assume the schema is valid. Set to true to assume the
-   * produced schema is valid.
-   *
-   * Default: false
-   */
-  assumeValid?: boolean,
-|};
```

Comment:
I still think it's better to have `type Options = {| ...GraphQLSchemaValidationOptions |}` even though there are no other options at the moment. It shows that `options` arg could be used for some other options beyond validation.

Reply:
Gotcha, makes sense 👍